easypost/easypost_client.py

```diff
@@ -33,7 +35,7 @@
 class EasyPostClient:
     """A client object used to authenticate and configure all HTTP calls to the EasyPost API."""
 
-    def __init__(self, api_key: str, api_base: str = API_BASE, timeout: int = TIMEOUT):
+    def __init__(self, api_key: str, api_base: str = f"{API_BASE}/{API_VERSION}", timeout: int = TIMEOUT):
```

Comment:
Nit: prefer to break this line of parameters

Reply:
Sure, can do in the next one